db-migrations/src/main/resources/db/migration/V5__nav_kontor_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE nav_kontor
+(
+    id       uuid PRIMARY KEY NOT NULL,
+    enhet_id varchar UNIQUE   NOT NULL,
+    navn     varchar          NOT NULL
+);
+
+ALTER TABLE bruker
+    ADD COLUMN nav_kontor_id uuid references nav_kontor (id);
```

Comment:
I teorien så kunne nav_kontor_id vært NOT NULL siden alle brukere skal ha kontor.
Vet ikke om det trengs å settes NOT NULL siden nav_kontor ikke er kritisk informasjon